ci/logging_tests_computes.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Prepare Logging Tests
+  # This setup needs to run on the controller, so that the expected
+  # log dirs exist and there won't be any errors copying files.
+  hosts: controller
+  gather_facts: true
+  vars_files:
+    - vars/common.yml
+  tasks:
+    - name: Create log dir
+      ansible.builtin.file:
+        path: "{{ logs_dir }}/logging_compute"
+        state: directory
+        mode: "0755"
+
+- name: "Verify logging journalctl identifiers"
+  hosts: computes
+  gather_facts: true
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    journal_test_id: "RHOSO-12681"
+    journal_list:
+      - ceilometer_agent_compute
+      - nova_compute
+  tasks:
+    - name: "Verify journalctl logging identifiers"
+      ansible.builtin.import_role:
+        name: telemetry_logging
+
+- name: Include report result
```

Comment:
when this run on compute node , it will not find the log dir and also we will need to update the report_result to copy the files from compute to controller